### ISSUE TITLE
Question about training config

### ISSUE THREAD

**Issue by cyysc1998** [opendatalab/FakeVLM#11]
Thanks for sharing the excellent work! I try to replicate the result in the paper but find some parameters are unclear. Firstly, the learning rate mentioned in the paper is 2e-4, whereas in the code it is 2e-5. Secondly, I noticed that the number of negative samples in the training set is significantly higher than that of positive samples. Could this lead to a biased model?

**Comment by lingcco** [comment]
Thank you for your question! Regarding the learning rate, please refer to the setting in our finetune.sh script (2e-5). The learning rate reported in the paper was incorrect, and we will issue a correction in future updates.
Concerning the class imbalance, it is true that in our dataset, the number of fake images exceeds that of real images. This imbalance could potentially have some impact on the model. However, based on our evaluation on other datasets, we have not observed significant bias caused by this imbalance. We will continue to monitor and further optimize sample balance in our future work.we sincerely appreciate your attention to our work and your valuable feedback！
